### PR TITLE
Cosmetic fixes

### PR DESCRIPTION
**Pull request by illright**
This PR will turn our app from a hot mess into an actually usable and pleasant-looking app. Compare with the current deployment on `main` and see for yourself

### CODE REVIEW

File: src/lib/widgets/now-playing/lib/lock-scroll-while-mounted.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export function lockScrollWhileMounted() {
+  document.body.style.overflow = 'hidden';
+
+  return () => {
+    document.body.style.overflow = 'visible';
```
Comment (aabounegm) [inline src/lib/widgets/now-playing/lib/lock-scroll-while-mounted.ts:5]: Why is it not `auto`?

**Review by khaledismaeel — COMMENTED** [review]
Good job!

File: src/lib/widgets/now-playing/ui/now-playing.svelte [hunk 2]
```diff
@@ -30,51 +27,73 @@
...
-  <div class="bg-slate-800 flex flex-col rounded-t-2xl py-10 px-8 items-center">
-    <div class="flex flex-col items-center relative">
-      {#if transcriptShown}
-        <Transcript {episode} class="absolute top-0 left-0 w-full h-full bg-slate-800 px-0.5" />
-      {/if}
-      <div class="flex flex-col items-center" class:invisible={transcriptShown}>
-        <EpisodeDisplay {episode} {podcast} />
+{#if $currentlyPlayingEpisode !== null}
+  <div
+    class="bg-black bg-opacity-20 flex flex-col justify-end fixed bottom-0 left-0 h-screen"
```
Comment (aabounegm) [inline src/lib/widgets/now-playing/ui/now-playing.svelte:36]: Is there a way to center it in the screen instead of aligning it to `left-0`?

**Review by aabounegm — COMMENTED** [review]
Thanks for the amazing effort, it looks a lot more pleasant now.
Is it possible to also remove the horizontal scroll-bar in the episode transcript?

File: src/lib/widgets/now-playing/lib/lock-scroll-while-mounted.ts (hunk 1 above)
Comment (illright) [reply src/lib/widgets/now-playing/lib/lock-scroll-while-mounted.ts:5]: That's the default, I believe

File: src/lib/widgets/now-playing/ui/now-playing.svelte (hunk 2 above)
Comment (illright) [reply src/lib/widgets/now-playing/ui/now-playing.svelte:36]: What for?

**Comment by illright** [comment]
@aabounegm this is by no means the last PR for cosmetics, so we'll come back to the changes you wanted at a later stage

File: src/lib/widgets/now-playing/ui/now-playing.svelte (hunk 2 above)
Comment (aabounegm) [reply src/lib/widgets/now-playing/ui/now-playing.svelte:36]: Because this looks ugly:
image

**Review by aabounegm — APPROVED** [review]

**Pull request merged**